ResourceStatusSystem/Client/ResourceStatusClient.py

```diff
@@ -50,16 +45,6 @@ class ResourceStatusClient( object ):
   the client considerably.
   """
 
-  def __init__( self ):
-    '''
-      The client tries to connect to :class:ResourceStatusDB by default. If it
-      fails, then tries to connect to the Service :class:ResourceStatusHandler.
-    '''
-
-    self.rssService = RPCClient( "ResourceStatus/ResourceStatus" )
-    self.validElements = RssConfiguration.getValidElements()
```

Comment:
This initialization is not needed any more ?

Reply:
Seems not to me.